src/PackageDependencyViewExtension/PackageDependencyViewExtension.cs

```diff
@@ -78,8 +78,11 @@ public void Startup(ViewStartupParams viewLoadedParams)
 
         public void Loaded(ViewLoadedParams viewLoadedParams)
         {
-            DependencyView = new PackageDependencyView(viewLoadedParams);
-            viewLoadedParams.AddToExtensionsSideBar(this, DependencyView);
+            DependencyView = new PackageDependencyView(this, viewLoadedParams);
+
+            if (DependencyView.MissingPackage) {
+                viewLoadedParams.AddToExtensionsSideBar(this, DependencyView);
```

Comment:
Dont think this line is needed, when the Loaded function gets called, it is almost always right after Dynamo initiation so you are always looking at a blank workspace, in that case the check will always be `false`. What do you think?

Reply:
yes, good point. we can move the API call here.